browser/src/control/Toolbar.js

```diff
@@ -395,6 +395,16 @@ L.Map.include({
 
 		if (this.uiManager.isUIBlocked())
 			return;
+
+		// This check is necessary to allow the user to change a language in the More... modal
+		if (this.jsdialog
+			&& this.jsdialog.hasDialogOpened()
+			&& command.startsWith('.uno:LanguageStatus?Language')
+		) {
+			app.socket.sendMessage('uno ' + command);
+			return;
+		}
+
 		if ((this.dialog.hasOpenedDialog() || (this.jsdialog && this.jsdialog.hasDialogOpened()))
 			&& !command.startsWith('.uno:ToolbarMode') && !force) {
```

Comment:
Here we added special parameter "force" for situation like this. Could you try use "force = true" in the source of uno command?

Reply:
its probably here: https://github.com/CollaboraOnline/online/blob/61aa932d3088ce698dab86304f8acaf86c78116d/browser/src/control/Control.LanguageDialog.js#L19

Reply:
@eszkadev Thanks for the quick reply. I noticed the `force` boolean but to pass it as `true` I need to pass also a null/empty json body because, as far I understand, the LanguageStatus command can be used only with query parameters:

https://github.com/CollaboraOnline/online/blob/32218cf54b862e2f422d7f17ea935628c2afbed6/browser/src/control/Toolbar.js#L343

https://github.com/CollaboraOnline/online/blob/61aa932d3088ce698dab86304f8acaf86c78116d/browser/src/control/Control.LanguageDialog.js#L20

Let me know if you prefer my approach or updating the PR passing an empty json and `true` for the force parameter

Reply:
null json is fine :)

Reply:
PR updated